docs_src/design/Process.md

```diff
@@ -52,20 +52,22 @@ Designs are created to address one or more requirements across one or more use c
 
 ## Project Board Cards and Issues
 
-All project design/architectural design decisions captured on the Design Decisions project board will be created as either a:
+All project design/architectural design decisions captured on the Design Decisions project board will be created as either:
 
-    Issue: for any design decision that will require code and a PR will be submitted against the issue.
-    Card: for any design decision that is not itself going to result in code or may need to be broken down into multiple issues (which can be referenced on the card).
+- Issue: for any design decision that will require code and a PR will be submitted against the issue.
+- Card: for any design decision that is not itself going to result in code or may need to be broken down into multiple issues (which can be referenced on the card).
 
 The template for project board cards documenting each decision is:
 
-    When/Where: date of the decision and place where the decision was made (such as TSC meeting, working group meeting, etc.). This section is required.
-    Decision Summary: quick write-up on the decision. This section is required.
-    Notes/Considerations: any alternatives discussed, any impacts to other decisions or considerations to be considered in the future (which would negate the decision). This section is optional.
+> **When/Where:** date of the decision and place where the decision was made (such as TSC meeting, working group meeting, etc.). This section is required.
+> 
+> **Decision Summary:** quick write-up on the decision. This section is required.
+> 
+> **Notes/Considerations:** any alternatives discussed, any impacts to other decisions or considerations to be considered in the future (which would negate the decision). This section is optional.
+> 
+> **Relevant links:** link to the meeting recording (if available). Link to ADR if relevant. Link to PRs or Issues if relevant. Required if available.
 
-    Relevant links: link to the meeting recording (if available). Link to ADR if relevant. Link to PRs or Issues if relevant. Required if available.
-
-Note there is a Template column on the project board with a single card that specifies this same structure.
+Note: there is a Template column on the project board with a single card that specifies this same structure.

```

Comment:
Are any of these and this whole section still relevant? I could not find such a Template column on project board? Which project board (we have [several](https://github.com/orgs/edgexfoundry/projects))?

Reply:
This is the project board being referenced here: https://github.com/orgs/edgexfoundry/projects/45 
Should add a link to it.